react-native/react/native/pinentry-listeners.desktop.js

```diff
@@ -0,0 +1,36 @@
+export default {
+  'keybase.1.secretUi.getPassphrase': (payload, response) => {
+    console.log('Asked for passphrase')
+    /*
+    Payload looks like:
+    { pinentry:
+      { features:
+        { secretStorage:
+          { allow: true, label: 'store your test passphrase' }
+        },
+        prompt: 'Enter a test passphrase',
+        retryLabel: '',
+        windowTitle: 'Keybase Test Passphrase' },
+        sessionID: 0
+      }
+    }
+    */
+    if (!('pinentry' in payload)) {
+      console.error('Passphrase payload has no pinentry object.')
+    } else if (!('features' in payload.pinentry)) {
+      console.error('payload.pinentry.features does not exist')
+    } else if (!('prompt' in payload.pinentry)) {
+      console.error('payload.pinentry.prompt does not exist')
```

Comment:
Not that I've seen so far!  But I wouldn't want to crash the main thread if there was.  We could just flow-annotate it, if we trust the protocol guarantees?